scintillometry/io/psrfits/tests/test_psrfits_read.py

```diff
@@ -9,7 +9,7 @@
 from astropy.coordinates import Longitude, Latitude, EarthLocation
 from astropy.time import Time
 
-from ..io import psrfits
+from ..core import open as fitsopen
```

Comment:
I think I'd just do `from ... import psrfits` and keep the file the same as it was otherwise